vpr/src/power/power.cpp

```diff
@@ -793,10 +793,10 @@ static void power_usage_routing(t_power_usage* power_usage,
     power_ctx.commonly_used->total_cb_buffer_size = 0.;
 
     /* Reset rr graph net indices */
-    for (size_t rr_node_idx = 0; rr_node_idx < device_ctx.rr_nodes.size(); rr_node_idx++) {
-        rr_node_power[rr_node_idx].net_num = ClusterNetId::INVALID();
-        rr_node_power[rr_node_idx].num_inputs = 0;
-        rr_node_power[rr_node_idx].selected_input = 0;
+    for (const RRNodeId& id : device_ctx.rr_graph.nodes()) {
+        rr_node_power[(size_t)id].net_num = ClusterNetId::INVALID();
```

Comment:
I prefer a name that makes it clear this id is for an rr_node (e.g. rr_id or rr_idx).  Not necessary to fix in these as it's not a critical style thing (but fine if you want to fix it), but I'd prefer that convention in future code (use an id name that indicates what kind of id it is).  Otherwise it can get hard to read code with multiple types of interacting id types (e.g. rr_idx and net_idx will be a lot easier to figure out than id and id2).

Reply:
I agree. I will change the variable name as you suggested.